department-of-veterans-affairs/va.gov-cms
# VACMS-5776: Add char count to featured content paragraphs.

## Description
See #5776

## Testing done
Visual / Behat

## Screenshots
![image](https://user-images.githubusercontent.com/2404547/125303364-be756c80-e2fa-11eb-9dcc-c57af5aaada4.png)

## QA steps
- As an administrator, go to `/node/3792/edit` and open up one of the featured content paragraphs.
- Visually verify the text counter.

### Definition of Done


### Select Team for PR review


### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
